api/nodes/serializers.py

```diff
@@ -124,9 +128,126 @@ def update(self, instance, validated_data):
         return instance
 
 
+class NodeContributorsSerializer(JSONAPISerializer):
+    """ Separate from UserSerializer due to necessity to override almost every field as read only
+    """
+    filterable_fields = frozenset([
+        'fullname',
+        'given_name',
+        'middle_name',
+        'family_name',
+        'id',
+        'bibliographic',
+        'permissions'
+    ])
+    id = ser.CharField(source='_id', label='ID')
+    fullname = ser.CharField(read_only=True, help_text='Display name used in the general user interface')
```

Comment:
This is how the field is defined on our user model, and seems to me a matter of semantic preference.

I'd be willing to hear thoughts on why this should be changed.